idea/resources-fir/META-INF/plugin.xml

```diff
@@ -240,6 +242,15 @@ The Kotlin FIR plugin provides language support in IntelliJ IDEA and Android Stu
     <projectService serviceInterface="org.jetbrains.kotlin.asJava.LightClassGenerationSupport"
                     serviceImplementation="org.jetbrains.kotlin.idea.asJava.FirLightClassGenerationSupport"/>
 
+    <projectService serviceInterface="org.jetbrains.kotlin.resolve.CodeAnalyzerInitializer"
```

Comment:
This services are parts of FE10 plugin and should not be present in FIR plugin

Reply:
Thanks! I have changed the test setup code to use FIR diagnostics instead.

Reply:
Also, removing FE1.0 makes a previous crashing test working! 😁